server/src/controllers/index.js

```diff
@@ -3,11 +3,13 @@ const { signIn } = require('./users');
 
 const { handleAddUser } = require('./users');
 const { signUpSchema } = require('./validations');
+const { productDetails } = require('./products/index');
```

Comment:
you can remove the index from the path
```suggestion
const { productDetails } = require('./products');
```